docs/docs/desktop/overview.md

```diff
@@ -0,0 +1,25 @@
+# Mockzilla Desktop
+
+The Mockzilla desktop app can be downloaded [here](https://mockzilla-test.pages.dev/).
+
+It allows manipulation of the mock data being returned by Mockzilla on your device.
+
+!!! important
+    You **must** be using the same Wifi network on your device running Mockzilla and the device running
+    the destkop app.
+
+### (1): Connect your device
+
+Your device should be automatically discovered by Mockzilla (you may need to restart the app on your device).
+
+
+![alt text](img/device_connection.png "Device connection")
+
+!!! note
+    If network discovery does find your device you can manually type in the IP address of your device. (Don't forget the port!)
```

Comment:
Should this say 'doesn't find your device'?

Reply:
Also I've been able to connect to an emulator by typing `:60000` without giving the port